src/pages/pricing-plans/options.ts

```diff
@@ -0,0 +1,52 @@
+export type PricingPlans = {
+  title: string
+  model: string
+  badges?: string[]
+  description: string
+  price: number
+  features: Feature[]
+  class?: string
+}
+
+type Feature = {
+  description: string
+  isAvailable: boolean
+}
+
+const features = [
+  'Up to 10 Active Users',
+  'Up to 30 Project Integrations',
+  'Analytics Module',
+  'Finance Module',
+  'Accounting Module',
+  'Network Platform',
+  'Unlimited Cloud Spase',
+]
+
+export const pricingPlans: PricingPlans[] = [
+  {
+    title: 'Startup',
+    model: 'Startup',
+    description: 'Optimal for 10+ team size and new startup',
+    price: 39,
+    features: features.map((d, i) => ({ description: d, isAvailable: i < 3 })),
+    class: '!bg-[#F7F9F9]',
```

Comment:
Don't we have similar colors in vuestic? (would be ideal to either use existing or have a follow up on this)

Reply:
there's no such color